frontend/dashboard/components/RepoList/ActionLinks.module.css

```diff
@@ -5,14 +5,21 @@
 }
 
 .giteaButton {
-  margin-right: 0.7rem;
+  margin-right: 0.45rem;
```

Comment:
Maybe we can use designsystemet tokens instead of hardcoded number? 😄 
I can see that `--fds-sizing-2` is 0.5 rem.

Reply:
Fixed 😊